lib/controllers/messages.ts

```diff
@@ -474,27 +473,14 @@ export function CreateMessageHandler(
     //
     // emit created message event to the output queue
     //
-    const errorOrNewMessageWithContent = NewMessageWithContent.decode({
```

Comment:
I suspect this decode here was needed due to some default fields that the decode() function would set when empty. If you get rid of this, default values won't be set.

Reply:
I could do a `CreatedMessageEvent.decode`, it should be equivalent as it's composed of the same sub-types as  `NewMessageWithContent`, what do you think? I'm fixing the tests now, perhaps the issue will emerge there

Reply:
that's right

Reply:
just found the problem with a test not passing, I confirm that using `CreatedMessageEvent.decode` fixes the problem